rollouts/controllers/remoterootsync_controller.go

```diff
@@ -136,42 +141,81 @@ func (r *RemoteRootSyncReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 
 	gkeCluster, err := r.store.GetCluster(ctx, cr.Name)
 	if err != nil {
+		r.updateStatus(ctx, &remoterootsync, clusterConnectivityFailed, nil)
```

Comment:
I need to double check the underlying code but just getting the cluster shouldn't require reaching out to the target cluster so `connectivityFailure` may not be applicable.